invokeai/backend/model_manager/load/model_loaders/flux.py

```diff
@@ -193,7 +198,30 @@ def _load_from_singlefile(
             sd = load_file(model_path)
             if "model.diffusion_model.double_blocks.0.img_attn.norm.key_norm.scale" in sd:
                 sd = convert_bundle_to_flux_transformer_checkpoint(sd)
+            futures: list[torch.jit.Future[tuple[str, torch.Tensor]]] = []
+            # For the first iteration we are just requesting the current size of the state dict
+            # This is due to an expected doubling of the tensor sizes in memory after converting float8 -> float16
+            # This should be refined in the future if not removed entirely when we support more data types
+            sd_size = asizeof.asizeof(sd)
+            cache_updated = False
+            for k in sd.keys():
+                v = sd[k]
+                if v.dtype != torch.bfloat16:
+                    if not cache_updated:
+                        self._ram_cache.make_room(sd_size)
```

Comment:
When making room in the cache we need to pass in the expected size of the state_dict _after_ casting, not the current size of the state_dict.